index.Rmd

```diff
@@ -103,15 +103,27 @@ You will be asked to give your credentials (username + password) and then the (a
 
 ### What to expect while running a Juno pipeline {#what-to-expect}  
 
-Our pipelines normally run by themselves. Sometimes they ask for input from the user to agree on installing software or a database but other than that, they do not require much input from the user. Try to read at least the last lines appearing on the screen and if you get asked to give a permission, please do. The first time you run a new pipeline, the preparation might take longer than expected. This is due to the installation of the basic software needed for it to work. Be patient! The installation may take some time.  
+The Juno pipelines usually run automated. Minimum input of the user is required. Detailed information on the required input can be found in the section for the specific pipeline. Sometimes a pipeline can ask for input from the user to agree on installing software or a database ([Figure 1.5](#installation-fig) step 1), if you get asked to give permission, please do so in order to run the pipeline. This is most often in the form of pressing y followed by enter ([Figure 1.5](#installation-fig) step 2).
```

Comment:
@wolthuisr About the text: "The Juno pipelines usually run automatically...". I like the screenshot but maybe you could divide it in two? I mean one up to the moment where you have to answer 'y' and the other one where the rest appears. Optionally, we can make a GIF out of it. For that you just need three pictures (one per step) and there are some websites where we can easily make it. I love GIFs but for tutorials sometimes they work and sometimes they don't so let me know what you think here.

Reply:
@AleSR13 I fixed the screenshots, added : for the notes, an example of a team page is visible in our slack chat!